duffy/worker.py

```diff
@@ -0,0 +1,94 @@
+'''
+<TODO>
+This is a [WIP] python file to update worker from v1.
+'''
+
+from config import ProdConfig
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+import json
+import subprocess
+
+import beanstalkc
+bs_obj = beanstalkc.Connection(host='127.0.0.1', parse_yaml=False)
+bs_obj.watch('requests')
+bs_obj.ignore('default')
+
+app = Flask(__name__)
```

Comment:
Let's try importing the app

`from duffy import APP` and use the sqlalchemy engine and configs from that object